src/sys/windows/tcp.rs

```diff
@@ -1,14 +1,14 @@
 use std::io;
 use std::mem::size_of;
-use std::net::{self, SocketAddr, SocketAddrV4, SocketAddrV6};
+use std::net::{self, Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV6};
 use std::time::Duration;
 use std::convert::TryInto;
 use std::os::windows::io::FromRawSocket;
 use std::os::windows::raw::SOCKET as StdSocket; // winapi uses usize, stdlib uses u32/u64.
 
 use winapi::ctypes::{c_char, c_int, c_ushort};
 use winapi::shared::ws2def::{SOCKADDR_STORAGE, AF_INET, SOCKADDR_IN};
-use winapi::shared::ws2ipdef::SOCKADDR_IN6_LH;
+use winapi::shared::ws2ipdef::SOCKADDR_IN6;
```

Comment:
After I had bumped `winapi` to `0.3.9` I realized this code already used `SOCKADDR_IN6_LH`. Would you prefer I use that type instead of this alias that was introduced in `0.3.9`?

Reply:
According to https://docs.rs/winapi/0.3.9/winapi/shared/ws2ipdef/type.SOCKADDR_IN6.html, `SOCKADDR_IN6` is a type alias for `SOCKADDR_IN6_LH`, so it's the same. I don't really care to be honest.

Reply:
Although I coudn't find any docs of `SOCKADDR_IN6`, Windows's docs about `SOCKADDR_IN6_LH` pops up as first results

Reply:
Ok. Let's revert to `SOCKADDR_IN6_LH`. Will make the PR cause less fuzz and risk being harder to backport or adopt for people or whatever.